packages/@react-spectrum/searchfield/docs/SearchAutocomplete.mdx

```diff
@@ -0,0 +1,484 @@
+Similar to [ComboBox](ComboBox.html), SearchAutocomplete accepts `<Item>` elements as children, each with a `key` prop. Basic usage of SearchAutocomplete, seen in the example above, shows multiple options populated with a string.
+Static collections, as in this example, can be used when the full list of options is known ahead of time.
+
+Dynamic collections, as shown below, can be used when the options come from an external data source such as an API call, or update over time.
+Providing the data in this way allows SearchAutocomplete to automatically cache the rendering of each item, which dramatically improves performance.
+
+As seen below, an iterable list of options is passed to the SearchAutocomplete using the `defaultItems` prop. 
+
+```tsx example
+function Example() {
+  let options = [
+    {id: 1, name: 'Aerospace'},
+    {id: 2, name: 'Mechanical'},
+    {id: 3, name: 'Civil'},
+    {id: 4, name: 'Biomedical'},
+    {id: 5, name: 'Nuclear'},
+    {id: 6, name: 'Industrial'},
+    {id: 7, name: 'Chemical'},
+    {id: 8, name: 'Agricultural'},
+    {id: 9, name: 'Electrical'}
+  ];
+
+  return (
+    <SearchAutocomplete
+      label="Search engineering majors"
+      defaultItems={options}>
+      {item => <Item>{item.name}</Item>}
+    </SearchAutocomplete>
+  );
+}
+```
+
+Alternatively, passing your list of options to SearchAutocomplete's `items` prop will cause the list of items to be controlled, useful for when you want to provide your own
+filtering logic. See the [Custom Filtering](#custom-filtering) section for more detail.
+
+### Internationalization
+To internationalize a SearchAutocomplete, a localized string should be passed to the `children` of each `Item`.
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of the SearchAutocomplete is automatically flipped.
+
+### Placeholder
+
+Placeholder text that describes the expected value or formatting for the SearchAutocomplete can be provided using the `placeholder` prop.
+Placeholder text will only appear when the SearchAutocomplete is empty, and should not be used as a substitute for labeling the component with a visible label.
+
+```tsx example
+function Example() {
+  let options = [
+    {id: 1, name: 'Aerospace'},
+    {id: 2, name: 'Mechanical'},
+    {id: 3, name: 'Civil'},
+    {id: 4, name: 'Biomedical'},
+    {id: 5, name: 'Nuclear'},
+    {id: 6, name: 'Industrial'},
+    {id: 7, name: 'Chemical'},
+    {id: 8, name: 'Agricultural'},
+    {id: 9, name: 'Electrical'}
+  ];
+
+  return (
+    <SearchAutocomplete
+      label="Search engineering majors"
+      defaultItems={options}
+      placeholder="Aerospace, Mechanical, etc.">
+      {item => <Item>{item.name}</Item>}
+    </SearchAutocomplete>
+  );
+}
+```
+
+## Labeling
+SearchAutocomplete can be labeled using the `label` prop. If the SearchAutocomplete is a required field, the `isRequired` and `necessityIndicator` props can be used to show a required state.
```

Comment:
Left this in because we should probably support whatever SearchField does, and it currently supports these. I'm open to removing these though; you definitely have a good point. I think we had a similar discussion for SearchWithin and didn't end up removing it.

Reply:
Added to follow-up (#2286)